[2025-02-07T22:42:04.091+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T22:42:04.218+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [queued]>
[2025-02-07T22:42:04.287+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [queued]>
[2025-02-07T22:42:04.288+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-07T22:42:05.218+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_directory> on 2025-02-07 03:00:00+00:00
[2025-02-07T22:42:05.323+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=20014) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T22:42:05.406+0000] {standard_task_runner.py:72} INFO - Started process 20046 to run task
[2025-02-07T22:42:06.015+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_directory', 'scheduled__2025-02-07T03:00:00+00:00', '--job-id', '1044', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmp7x4xs1se']
[2025-02-07T22:42:06.031+0000] {standard_task_runner.py:105} INFO - Job 1044: Subtask check_directory
[2025-02-07T22:42:07.160+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [running]> on host 003b7fbea265
[2025-02-07T22:42:08.437+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-07T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-07T03:00:00+00:00'
[2025-02-07T22:42:08.453+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T22:42:08.456+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T22:42:08.458+0000] {logging_mixin.py:190} INFO - Current task name:check_directory state:running start_date:2025-02-07 22:42:04.224774+00:00
[2025-02-07T22:42:08.460+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-07T22:42:08.462+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T22:42:08.468+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-07T22:42:08.480+0000] {logging_mixin.py:190} INFO - borusss ['in', 'logs', 'out', 'statu_date_execution.txt', 'succes_date_execution.txt', 'test', 'Écritures - Copie.csv.bak', 'Écritures comptables article.csv.bak', 'Écritures.csv.bak']
[2025-02-07T22:42:08.508+0000] {python.py:240} INFO - Done. Returned value was: check_file_in_directory
[2025-02-07T22:42:08.514+0000] {branch.py:38} INFO - Branch into check_file_in_directory
[2025-02-07T22:42:08.515+0000] {skipmixin.py:230} INFO - Following branch ('check_file_in_directory',)
[2025-02-07T22:42:08.632+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-07T22:42:08.905+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T22:42:08.910+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_directory, run_id=scheduled__2025-02-07T03:00:00+00:00, execution_date=20250207T030000, start_date=20250207T224204, end_date=20250207T224208
[2025-02-07T22:42:09.011+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T22:42:09.022+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T22:42:09.023+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-07 22:41:56.018789+00:00
[2025-02-07T22:42:09.024+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-07T22:42:09.113+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T22:42:09.175+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-07T22:43:34.719+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T22:43:34.861+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [queued]>
[2025-02-07T22:43:34.923+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [queued]>
[2025-02-07T22:43:34.924+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-07T22:43:35.855+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_directory> on 2025-02-07 03:00:00+00:00
[2025-02-07T22:43:35.905+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=20212) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T22:43:35.917+0000] {standard_task_runner.py:72} INFO - Started process 20251 to run task
[2025-02-07T22:43:35.928+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_directory', 'scheduled__2025-02-07T03:00:00+00:00', '--job-id', '1090', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpjnjzddg8']
[2025-02-07T22:43:36.007+0000] {standard_task_runner.py:105} INFO - Job 1090: Subtask check_directory
[2025-02-07T22:43:36.336+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [running]> on host 003b7fbea265
[2025-02-07T22:43:37.113+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-07T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-07T03:00:00+00:00'
[2025-02-07T22:43:37.127+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T22:43:37.133+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T22:43:37.139+0000] {logging_mixin.py:190} INFO - Current task name:check_directory state:running start_date:2025-02-07 22:43:34.884013+00:00
[2025-02-07T22:43:37.141+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-07T22:43:37.146+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T22:43:37.154+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-07T22:43:37.157+0000] {logging_mixin.py:190} INFO - borusss ['in', 'logs', 'out', 'statu_date_execution.txt', 'succes_date_execution.txt', 'test', 'Écritures - Copie.csv.bak', 'Écritures comptables article.csv.bak', 'Écritures.csv.bak']
[2025-02-07T22:43:37.162+0000] {python.py:240} INFO - Done. Returned value was: check_file_in_directory
[2025-02-07T22:43:37.170+0000] {branch.py:38} INFO - Branch into check_file_in_directory
[2025-02-07T22:43:37.178+0000] {skipmixin.py:230} INFO - Following branch ('check_file_in_directory',)
[2025-02-07T22:43:37.400+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-07T22:43:37.494+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T22:43:37.499+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_directory, run_id=scheduled__2025-02-07T03:00:00+00:00, execution_date=20250207T030000, start_date=20250207T224334, end_date=20250207T224337
[2025-02-07T22:43:37.535+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T22:43:37.545+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T22:43:37.546+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-07 22:43:27.245678+00:00
[2025-02-07T22:43:37.547+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-07T22:43:37.619+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T22:43:37.892+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-07T22:43:37.901+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-07T23:11:30.166+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T23:11:30.465+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [queued]>
[2025-02-07T23:11:30.559+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [queued]>
[2025-02-07T23:11:30.579+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-07T23:11:32.005+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_directory> on 2025-02-07 03:00:00+00:00
[2025-02-07T23:11:32.021+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=20768) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T23:11:32.042+0000] {standard_task_runner.py:72} INFO - Started process 20800 to run task
[2025-02-07T23:11:32.048+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_directory', 'scheduled__2025-02-07T03:00:00+00:00', '--job-id', '1139', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpaotjng2u']
[2025-02-07T23:11:32.052+0000] {standard_task_runner.py:105} INFO - Job 1139: Subtask check_directory
[2025-02-07T23:11:32.599+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [running]> on host 003b7fbea265
[2025-02-07T23:11:33.094+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-07T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-07T03:00:00+00:00'
[2025-02-07T23:11:33.099+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T23:11:33.103+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T23:11:33.104+0000] {logging_mixin.py:190} INFO - Current task name:check_directory state:running start_date:2025-02-07 23:11:30.492392+00:00
[2025-02-07T23:11:33.105+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-07T23:11:33.107+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T23:11:33.110+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-07T23:11:33.113+0000] {logging_mixin.py:190} INFO - borusss ['in', 'logs', 'out', 'statu_date_execution.txt', 'succes_date_execution.txt', 'test', 'Écritures - Copie.csv.bak', 'Écritures comptables article.csv.bak', 'Écritures.csv.bak']
[2025-02-07T23:11:33.117+0000] {python.py:240} INFO - Done. Returned value was: check_file_in_directory
[2025-02-07T23:11:33.119+0000] {branch.py:38} INFO - Branch into check_file_in_directory
[2025-02-07T23:11:33.121+0000] {skipmixin.py:230} INFO - Following branch ('check_file_in_directory',)
[2025-02-07T23:11:33.204+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-07T23:11:33.461+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T23:11:33.470+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_directory, run_id=scheduled__2025-02-07T03:00:00+00:00, execution_date=20250207T030000, start_date=20250207T231130, end_date=20250207T231133
[2025-02-07T23:11:33.820+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T23:11:33.832+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T23:11:33.838+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-07 23:11:21.033405+00:00
[2025-02-07T23:11:33.846+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-07T23:11:34.030+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T23:11:34.309+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-07T23:11:34.479+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-07T23:15:02.839+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-07T23:15:03.031+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [queued]>
[2025-02-07T23:15:03.098+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [queued]>
[2025-02-07T23:15:03.106+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-07T23:15:07.180+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_directory> on 2025-02-07 03:00:00+00:00
[2025-02-07T23:15:07.265+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_directory', 'scheduled__2025-02-07T03:00:00+00:00', '--job-id', '1187', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpcsliqpnh']
[2025-02-07T23:15:07.271+0000] {standard_task_runner.py:105} INFO - Job 1187: Subtask check_directory
[2025-02-07T23:15:07.242+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=87) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-07T23:15:07.274+0000] {standard_task_runner.py:72} INFO - Started process 129 to run task
[2025-02-07T23:15:08.318+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_directory scheduled__2025-02-07T03:00:00+00:00 [running]> on host 003b7fbea265
[2025-02-07T23:15:09.008+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-07T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-07T03:00:00+00:00'
[2025-02-07T23:15:09.016+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-07T23:15:09.017+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-07T23:15:09.018+0000] {logging_mixin.py:190} INFO - Current task name:check_directory state:running start_date:2025-02-07 23:15:03.066375+00:00
[2025-02-07T23:15:09.019+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-07T23:15:09.020+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-07T23:15:09.025+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-07T23:15:09.035+0000] {logging_mixin.py:190} INFO - borusss ['in', 'logs', 'out', 'statu_date_execution.txt', 'succes_date_execution.txt', 'test', 'Écritures - Copie.csv.bak', 'Écritures comptables article.csv.bak', 'Écritures.csv.bak']
[2025-02-07T23:15:09.050+0000] {python.py:240} INFO - Done. Returned value was: check_file_in_directory
[2025-02-07T23:15:09.053+0000] {branch.py:38} INFO - Branch into check_file_in_directory
[2025-02-07T23:15:09.055+0000] {skipmixin.py:230} INFO - Following branch ('check_file_in_directory',)
[2025-02-07T23:15:09.187+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-07T23:15:09.406+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-07T23:15:09.409+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_directory, run_id=scheduled__2025-02-07T03:00:00+00:00, execution_date=20250207T030000, start_date=20250207T231503, end_date=20250207T231509
[2025-02-07T23:15:09.525+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-07T23:15:09.531+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-07T23:15:09.532+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-07 23:14:41.987446+00:00
[2025-02-07T23:15:09.533+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-07T23:15:09.677+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-07T23:15:09.752+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
